src/java/src/main/java/com/dnanexus/DXProject.java

```diff
@@ -219,6 +223,458 @@ public void destroy(boolean terminateJobs) {
                 RetryStrategy.SAFE_TO_RETRY);
     }
 
+    /**
+     * Move specified files or folder to an archive storage to save on storage costs.
```

Comment:
Can you reword this to "Request specified files or folder to be archived"

Reply:
done